### DISCUSSION TITLE
github SSL certi ficate problem: unable to get local issuer certificate

### DISCUSSION THREAD

**Discussion by LeolandGuo** [hexojs/hexo#5501] [Question / Answer] [answered]
安装 Hexo 完成后，请执行下列命令，Hexo 将会在指定文件夹中新建所需要的文件。 $ hexo init <folder>输入时出现：$ hexo init website
INFO  Validating config
INFO  Cloning hexo-starter https://github.com/hexojs/hexo-starter.git
fatal: unable to access 'https://github.com/hexojs/hexo-starter.git/': SSL certi
ficate problem: unable to get local issuer certificate
WARN  git clone failed. Copying data instead
INFO  Install dependencies
INFO  Start blogging with Hexo!

**Comment by renbaoshuo** [comment, marked as answer]
请检查您的网络连接，确保能够正常访问 GitHub。

**Comment by uiolee** [comment]
```
WARN git clone failed. Copying data instead
```

This is no problem for `hexo init`, it already does its job using fallback methods.

----------------------
----------------------

```
fatal: unable to access 'https://github.com/hexojs/hexo-starter.git/': SSL certi
ficate problem: unable to get local issuer certificate
```
For `git`, the logs already tell you the problem, and it's caused by the ssl certificate. This is probably a well-known network problem for Chinese users.
